l10n tests failing due to conversion to babel@7

See https://circleci.com/gh/mozilla/fxa-content-server-l10n/4283

> { Error: Cannot find module 'babel-plugin-transform-class-properties' from '/home/circleci/project/workspace/fxa/packages/fxa-content-server'
    at Function.module.exports [as sync] (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/resolve/lib/sync.js:58:15)
    at resolveStandardizedName (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at items.map (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at alias (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:63:49) code: 'MODULE_NOT_FOUND' }
Warning: Command failed: node_modules/.bin/babel --plugins=babel-plugin-syntax-dynamic-import,dynamic-import-webpack,transform-class-properties --presets es2015 app/scripts --out-dir .es5
{ Error: Cannot find module 'babel-plugin-transform-class-properties' from '/home/circleci/project/workspace/fxa/packages/fxa-content-server'
    at Function.module.exports [as sync] (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/resolve/lib/sync.js:58:15)
    at resolveStandardizedName (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at items.map (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at alias (/home/circleci/project/workspace/fxa/packages/fxa-content-server/node_modules/@babel/core/lib/config/config-descriptors.js:63:49) code: 'MODULE_NOT_FOUND' }

I forgot to update https://github.com/mozilla/fxa/blob/fa2fa7a90d11fac965c28cace16ea1a6fd01156d/packages/fxa-content-server/grunttasks/l10n-extract.js#L15

--- a/packages/fxa-content-server/grunttasks/l10n-extract.js
+++ b/packages/fxa-content-server/grunttasks/l10n-extract.js
@@ -12,7 +12,7 @@ var execSync = require('child_process').execSync;
 
 // where to place the pot files.
 var messagesOutputPath = path.join(__dirname, '..', 'locale', 'templates', 'LC_MESSAGES');
-var babelCmd = 'node_modules/.bin/babel --plugins=babel-plugin-syntax-dynamic-import,dynamic-import-webpack,transform-class-properties --presets es2015 app/scripts --out-dir .es5';
+var babelCmd = 'npx babel --plugins=@babel/plugin-syntax-dynamic-import,dynamic-import-webpack,@babel/plugin-proposal-class-properties --presets @babel/env app/scripts --out-dir .es5';
 var templateCmd = 'cp -r app/scripts/templates .es5/templates/';
 
 module.exports = function (grunt) {

